modules/olm-updating-index-image.adoc

```diff
@@ -39,6 +57,22 @@ $ opm index add \
 $ podman push <registry>/<namespace>/<existing_index_image>:<tag>
 ----
 
+ifeval::["{context}" == "olm-restricted-networks"]
+. Follow the steps in the _Mirroring an Operator catalog_ procedure again to mirror the updated content. However, when you get to the step about creating the `ImageContentSourcePolicy` (ICSP) object, use the `oc replace` command instead of the `oc create` command. For example:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc replace -f ./{index-image}-manifests/imageContentSourcePolicy.yaml
+----
++
+This change is required because the object already exists and must be updated.
++
+[NOTE]
+====
+Normally, the `oc apply` command can be used to update existing objects that were previously created using `oc apply`. However, due to a known issue regarding the size of the `metadata.annotations` field in ICSP objects, the `oc replace` command must be used for this step currently. _(Link to BZ.)_
```

Comment:
Placeholder "link".

Reply:
Just flagging that the BZ is still not attached. :slightly_smiling_face:

Reply:
@kevinrizza Do we want to mention a BZ here?